src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+
+    if (secondCell && lastCell) {
+      const clonedCell = secondCell.cloneNode(true);
+
+      row.insertBefore(clonedCell, lastCell);
+    }
+  });
+}
+
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const tfoot = document.querySelector('tfoot');
```

Comment:
It's better to check if the elements actually exist before passing them to the `cloneColumn` function. If any of these selectors don't match any element in the document, the function will throw an error.